WordPress/WordPressHomeWidgetToday/Model/TodayWidgetContent.swift

```diff
@@ -0,0 +1,12 @@
+import WidgetKit
+
+
+struct TodayWidgetContent: TimelineEntry {
+    var date = Date()
```

Comment:
Is there a reason these properties couldn't be `let`s?

Reply:
Nope. Fixed